templates/content/field--oe-document-reference--oe-document--oe-document.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
```

Comment:
why not field--oe-document-reference--oe-document.html.twig ?

Reply:
@drishu Why?
The original name targets entity type + field name + bundle. Reducing it would only target entity type + field name or entity type + bundle. Or worse, in this case I think it will target both. So it will target other fields in oe_document view mode, _and_ it will target other instances of the oe_document field.

I think we should usually target the exact field instance. And for nodes, we should target the field instance in a specific view mode.

@escuriola ^